content/reference-capabilities/capability-subtyping.md

```diff
@@ -8,39 +8,53 @@ menu:
+`val` can't become `ref` since it can't be used to write (and there could be other `val` aliases requiring immutability).
 
-Now let's consider what happens when we have an alias of a reference capability. For example, if we have some `iso` and we alias it (without doing a `consume` or a destructive read), the type we get is `iso!`, not `iso`.
+Also keep in mind, subtyping is _transitive_. That means that since `val <: box` and `box <: tag`, we also get `val <: tag`.
 
-* `iso! <: tag`. This is a pretty big change. Instead of being a subtype of everything like `iso`, the only thing an `iso!` is a subtype of is `tag`. This is because the `iso` still exists, and is still _read and write unique_. Any alias can neither read from nor write to the object. That means an `iso!` can only be a subtype of `tag`.
-* `trn! <: box`. This is a change too, but not as big a change. Since `trn` is only _write unique_, it's ok for aliases to read from the object, but it's not ok for aliases to write to the object. That means we could have `box` or `val` aliases - except `val` guarantees that _no_ alias can write to the object! Since our `trn` still exists and can write to the object, a `val` alias would break the guarantees that `val` makes. So a `trn!` can only be a subtype of `box` (and, transitively, `tag` as well).
-* `ref! <: ref`. Since a `ref` only guarantees that _other_ actors can neither read from nor write to the object, it's ok to make more `ref` aliases within the same actor.
-* `val! <: val`. Since a `val` only guarantees that _no_ actor can write to the object, its ok to make more `val` aliases since they can't write to the object either.
-* `box! <: box`. A `box` only guarantees that _other_ actors can't write to the object. Both `val` and `ref` make that guarantee too, so why can `box` only alias as `box`? It's because we can't make _more_ guarantees when we alias something. That means `box` can only alias as `box`.
-* `tag! <: tag`. A `tag` doesn't make any guarantees at all. Just like with a `box`, we can't make more guarantees when we make a new alias, so a `tag` can only alias as a `tag`.
+## Subtypes of unique capabilities
 
-## Ephemeral substitution
+When it comes to talking about unique capabilities, the situation is a bit more complex. With variables, we only had the six basic capabilities,
+but we're talking about expressions here. We'll have to work with aliased and unaliased forms of the capabilities.
 
-The last case to consider is when we have an ephemeral reference capability. For example, if we have some `iso` and we `consume` it or do a destructive read, the type we get is `iso^`, not `iso`.
+Let's get one thing settled right off the bat: `iso! = tag`, and `trn! = box`. Remember that `!` is the modifier for stable, named aliases. So it's the strongest capability that is compatible with the original. In the case of `iso`, nothing else is allowed access, so we get `tag`, and for `trn` we allow readable aliases,
```

Comment:
Good catch, do you have any good ideas for alternatives? Maybe just immediately?

Reply:
Good question. The concept being expressed here is that of `!` and how it interacts with `iso` and `trn` so either:

> First thing of importance: `iso! = tag` and `trn! = box`...

Or you could take a topic and stress positions focused approach, so shift the paragraph around to something like:

> Let us review the importance of `!` for capabilities. Remember that `!` is the modifier for stable, named aliases; so using it gives the strongest capability that is compatible with the original capability. In the case of `iso`, nothing else is allowed access, so with `iso!` we get `tag`, while for `trn` we allow readable aliases, so with `trn!` we get `box`. Meaning, `iso! = tag`, and `trn! = box`.

---

If you are wondering where I get what can sometimes seem like arbitrary writing rules, it is a combination of experience writing with/for Malagasy (Madagascar nationals) and following guides such as: https://www.springernature.com/gp/authors/campaigns/writing-in-english

Reply:
Thanks for the thoughts here. I'm thinking I might need to go and elaborate more on this in previous chapters as well; previously this subtyping chapter played a double role of explaining some aliasing and unaliasing as well as the subtyping relation itself. So in a bit of a winding way, I like the examples you've provided, but I'm thinking perhaps a bit of restructuring will be needed here. This has scaled up a bit past the idiom but the original reason I was writing that this way was related to pacing.

> If you are wondering where I get what can sometimes seem like arbitrary writing rules, it is a combination of experience writing with/for Malagasy (Madagascar nationals) and following guides such as: https://www.springernature.com/gp/authors/campaigns/writing-in-english

I have actually felt that the rules you've given have been sensible enough that they haven't felt arbitrary. But that resource will be quite useful to read through, many thanks for sharing

Reply:
So what seemed to work out here was  to add a tiny bit more in the aliasing types section to make the `!` uses make sense, then killing the aliasing bit from here since aliasing isn't really coupled to subtyping as much anymore.

Reply:
I think moving the aliasing made it easier to understand.